runtimevar/constantvar/example_test.go

```diff
@@ -88,3 +88,21 @@ func Example_error() {
 	// Output:
 	// runtimevar (code=Unknown): my error
 }
+
+func Example_openVariable() {
```

Comment:
Similar to #1302, it would likely be best to move this out to a separate example file.

Reply:
I'm doing that in a separate PR, although I'm only doing it for the portable API example (`runtimevar`, `blob`, etc.), not for every driver; I don't think the latter is necessary do you?

Reply:
I think it's nice to show the full code necessary for the example to work, but optional.